pulsar-common/src/main/java/com/yahoo/pulsar/common/util/collections/ConcurrentLongPairSet.java

```diff
@@ -528,4 +529,23 @@ public int compareTo(LongPair o) {
             }
         }
     }
+    
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append('{');
+        final AtomicBoolean first = new AtomicBoolean(true);
```

Comment:
well, unless the lambda body below can be run concurrently.  but in that case, the body doesn't work, so i'm assuming that isn't the case.